ui/lib/core/addon/components/form-field.hbs

```diff
@@ -40,6 +40,46 @@
             <G.Error data-test-field-validation={{this.valuePath}}>{{this.validationError}}</G.Error>
           {{/if}}
         </Hds::Form::Checkbox::Group>
+      {{else}}
+        <Hds::Form::Select::Field
+          name={{@attr.name}}
+          @id={{@attr.name}}
+          {{on "change" this.onChangeWithEvent}}
+          data-test-input={{@attr.name}}
+          @isInvalid={{this.validationError}}
```

Comment:
I have added this line so the border is red when invalid, per HDS specs

Reply:
What if the user is required to select a value from the list? Wouldn't this trigger a validation error (in which case, the red border around the select can help the user identify the error)

Reply:
There you go, that would be one. Thank you for pointing that out.